app/predacons_cli/src/cli.py

```
@@ -19,13 +19,32 @@ def launch(self):
+        model,tokenizer = Cli.load_model(self,config["model_path"],
+                                         config["trust_remote_code"],
+                                         config["use_fast_generation"],
+                                         config["draft_model_name"],
+                                         config["gguf_file"],
+                                         config["auto_quantize"])
         
-        # model,tokenizer = Cli.load_model(config.model_path,trust_remote_code,use_fast_generation,draft_model_name,gguf_file,auto_quantize)
+        chat  = []
+        while True:
+            user_input = Prompt.ask("Enter your query")
+            user_body = {"role": "user", "content": user_input} 
```

Ruff

Local variable `user_body` is assigned to but never used

Remove assignment to unused variable `user_body`

(F841)